--- ghostscript-9.09/base/lib.mak.0300~	2013-08-21 16:06:40.000000000 +0200
+++ ghostscript-9.09/base/lib.mak	2013-08-30 22:29:47.523498248 +0200
@@ -1015,7 +1015,7 @@ $(GLOBJ)gsparam.$(OBJ) : $(GLSRC)gsparam
 $(GLOBJ)gsparamx.$(OBJ) : $(AK) $(GLSRC)gsparamx.c $(string__h)\
  $(gserrors_h) $(gsmemory_h) $(gsparam_h) $(gsparamx_h)\
  $(gstypes_h) $(MAKEDIRS)
-	$(GLCC) $(GLO_)gsparamx.$(OBJ) $(C_) $(GLSRC)gsparamx.c
+	$(GLCCSHARED) $(GLO_)gsparamx.$(OBJ) $(C_) $(GLSRC)gsparamx.c
 
 # Future replacement for gsparams.c
 $(GLOBJ)gsparam2.$(OBJ) : $(GLSRC)gsparam2.c $(AK) $(gx_h)\
@@ -1293,7 +1293,7 @@ $(GLD)gxfapi.dev : $(LIB_MAK) $(ECHOGS_X
 
 # Provide a mapping between StandardEncoding and ISOLatin1Encoding.
 $(GLOBJ)gdevemap.$(OBJ) : $(GLSRC)gdevemap.c $(AK) $(std_h) $(MAKEDIRS)
-	$(GLCC) $(GLO_)gdevemap.$(OBJ) $(C_) $(GLSRC)gdevemap.c
+	$(GLCCSHARED) $(GLO_)gdevemap.$(OBJ) $(C_) $(GLSRC)gdevemap.c
 
 # ----------- Downsampling routines ------------ #
 gxdownscale_h=$(GLSRC)gxdownscale.h $(gsmemory_h) $(gxdevcli_h) $(ctype__h) \
--- ghostscript-9.09/base/unix-dll.mak.0300~	2013-08-30 22:29:47.523498248 +0200
+++ ghostscript-9.09/base/unix-dll.mak	2013-08-31 11:34:58.219078883 +0200
@@ -90,11 +90,11 @@ $(GS_SO_MAJOR): $(GS_SO_MAJOR_MINOR)
 
 # Build the small Ghostscript loaders, with Gtk+ and without
 $(GSSOC_XE): so-links-subtarget $(PSSRC)$(SOC_LOADER)
-	$(GLCC) -g -o $(GSSOC_XE) $(PSSRC)dxmainc.c \
+	$(GLCC) -g $(LDFLAGS) -o $(GSSOC_XE) $(PSSRC)dxmainc.c \
 	-L$(BINDIR) -l$(GS_SO_BASE)
 
 $(GSSOX_XE): so-links-subtarget $(PSSRC)$(SOC_LOADER)
-	$(GLCC) -g $(SOC_CFLAGS) -o $(GSSOX_XE) $(PSSRC)$(SOC_LOADER) \
+	$(GLCC) -g $(SOC_CFLAGS) $(LDFLAGS) -o $(GSSOX_XE) $(PSSRC)$(SOC_LOADER) \
 	-L$(BINDIR) -l$(GS_SO_BASE) $(SOC_LIBS)
 
 # ------------------------- Recursive make targets ------------------------- #
