--- ghostscript-9.09/psi/dxmain.c.omv~	2013-08-31 11:54:25.034971659 +0200
+++ ghostscript-9.09/psi/dxmain.c	2013-08-31 11:55:14.372052084 +0200
@@ -1160,7 +1160,7 @@ int main(int argc, char *argv[])
     use_gui = gtk_init_check(&argc, &argv);
 
     /* insert display device parameters as first arguments */
-    gs_sprintf(dformat, "-dDisplayFormat=%d",
+    sprintf(dformat, "-dDisplayFormat=%d",
             DISPLAY_COLORS_RGB | DISPLAY_ALPHA_NONE | DISPLAY_DEPTH_8 |
             DISPLAY_BIGENDIAN | DISPLAY_TOPFIRST);
     nargc = argc + 1;
