--- ghostscript-9.15/devices/devs.mak.0032~	2014-11-06 14:00:30.379685192 +0100
+++ ghostscript-9.15/devices/devs.mak	2014-11-06 14:00:50.838429422 +0100
@@ -1589,7 +1589,7 @@ $(DD)pngalpha.dev : $(DEVS_MAK) $(libpng
 fpng_=$(DEVOBJ)gdevfpng.$(OBJ) $(DEVOBJ)gdevpccm.$(OBJ)
 
 $(DEVOBJ)gdevfpng_0.$(OBJ) : $(DEVSRC)gdevfpng.c\
- $(gdevprn_h) $(gdevpccm_h) $(gscdefs_h) $(zlib_h)
+ $(gdevprn_h) $(gdevpccm_h) $(gscdefs_h)
 	$(CC_) $(I_)$(DEVI_) $(II)$(PI_)$(_I) $(PCF_) $(GLF_) $(DEVO_)gdevfpng_0.$(OBJ) $(C_) $(DEVSRC)gdevfpng.c
 
 $(DEVOBJ)gdevfpng_1.$(OBJ) : $(DEVSRC)gdevfpng.c\
