diff -up ghostpdl-9.51/Resource/Init/gs_fonts.ps.2~ ghostpdl-9.51/Resource/Init/gs_fonts.ps
--- ghostpdl-9.51/Resource/Init/gs_fonts.ps.2~	2020-03-15 20:37:43.696586167 +0100
+++ ghostpdl-9.51/Resource/Init/gs_fonts.ps	2020-03-15 20:39:35.711029531 +0100
@@ -139,10 +139,20 @@ userdict /.nativeFontmap .FontDirectory
       } if
       1 index type /stringtype eq
       1 index type /nametype eq and 1 index xcheck and
-      1 index /run eq 2 index /.runlibfile eq or and {
+      1 index /run eq 2 index /.runlibfile eq 3 index /.runlibfileifexists eq or or and {
                 % This is an inclusion entry.
-        pop findlibfile { exch pop } { file } ifelse
-        2 index exch .readFontmap pop
+        
+        0 index /.runlibfileifexists eq {
+          pop findlibfile {
+            exch pop
+            2 index exch .readFontmap pop
+          } {
+            pop
+          } ifelse
+        } {
+          pop findlibfile { exch pop } { file } ifelse
+          2 index exch .readFontmap pop
+        } ifelse
       } {
                 % This is a real entry.
                 % Read and pop tokens until a semicolon.
diff -up ghostpdl-9.51/Resource/Init/gs_init.ps.2~ ghostpdl-9.51/Resource/Init/gs_init.ps
--- ghostpdl-9.51/Resource/Init/gs_init.ps.2~	2020-03-12 12:49:11.000000000 +0100
+++ ghostpdl-9.51/Resource/Init/gs_init.ps	2020-03-15 20:40:10.829420099 +0100
@@ -863,6 +863,14 @@ currentdict /.makeinternaldict .undef
            { /undefinedfilename signalerror }
           ifelse
         } bind def
+/runlibfileifexists
+        {		% We don't want to bind 'run' into this procedure,
+                        % since run may get redefined.
+          findlibfile
+           { exch pop /run .systemvar exec }
+           { pop }
+          ifelse
+        } bind def
 /selectdevice
         { finddevice setdevice .setdefaultscreen } bind def
 /signalerror		% <object> <errorname> signalerror -
@@ -991,6 +999,7 @@ def
         } bind def
 % Temporarily substitute it for the real runlibfile.
 /.runlibfile /runlibfile load def
+/.runlibfileifexists /runlibfileifexists load def
 /runlibfile //runlibfile0 def
 currentdict /runlibfile0 .undef
 
