diff -up ghostpdl-10.03.0/configure.ac.omv~ ghostpdl-10.03.0/configure.ac
--- ghostpdl-10.03.0/configure.ac.omv~	2024-03-11 23:22:03.370743189 +0100
+++ ghostpdl-10.03.0/configure.ac	2024-03-11 23:23:41.255769619 +0100
@@ -752,9 +752,7 @@ SUBCONFIG_OPTS="--build=$build --host=$h
 dnl --------------------------------------------------
 dnl Check for NEON support
 dnl --------------------------------------------------
-save_cflags=$CFLAGS
 AC_MSG_CHECKING([neon support])
-CFLAGS="$save_cflags $OPT_CFLAGS -mfpu=neon -mcpu=cortex-a53"
 HAVE_NEON=""
 AC_LINK_IFELSE(
   [AC_LANG_PROGRAM([#include "arm_neon.h"], [
@@ -774,7 +772,6 @@ if test "x$HAVE_NEON" != x; then
 else
   AC_MSG_RESULT(no)
 fi
-CFLAGS=$save_cflags
 
 dnl --------------------------------------------------
 dnl Check for libraries
@@ -988,14 +985,13 @@ if test x$with_tesseract != xno; then
         fi
 
         AC_MSG_CHECKING([neon support])
-        CXXFLAGS="$save_cxxflags -mfpu=neon -mcpu=cortex-a53"
 
         TESS_NEON=""
         AC_LINK_IFELSE(
          [AC_LANG_PROGRAM([#include "arm_neon.h"],
                           [int32x4_t round = vdupq_n_s32(10);
                            return(0);])],
-         [TESS_NEON="-mfpu=neon -mcpu=cortex-a53 -D__ARM_NEON__"],
+         [TESS_NEON="-D__ARM_NEON__"],
          [TESS_NEON=""])
 
         if test "x$TESS_NEON" != x && test "x$enable_neon" != xno; then
@@ -1006,8 +1002,6 @@ if test x$with_tesseract != xno; then
           TESS_NEON=""
         fi
 
-        CXXFLAGS="$save_cxxflags"
-
         OCR_VERSION=1
         OCR_SHARED=0
         OCR_DEVS="\$(DD)ocr.dev \$(DD)hocr.dev \$(DD)pdfocr8.dev \$(DD)pdfocr24.dev \$(DD)pdfocr32.dev"
@@ -2432,7 +2426,7 @@ if test x$WITH_CAL != x0; then
     CAL_CFLAGS="$CAL_CFLAGS $HAVE_AVX2"
   fi
   if test "x$HAVE_NEON" != x; then
-    CAL_NEON_CFLAGS="-mfpu=neon -mcpu=cortex-a53"
+    CAL_NEON_CFLAGS=""
     CAL_CFLAGS="$CAL_CFLAGS $HAVE_NEON"
   fi
 fi
