diff -up ghostpdl-10.03.0/Resource/Init/gs_fonts.ps.2~ ghostpdl-10.03.0/Resource/Init/gs_fonts.ps
--- ghostpdl-10.03.0/Resource/Init/gs_fonts.ps.2~	2024-03-06 10:52:27.000000000 +0100
+++ ghostpdl-10.03.0/Resource/Init/gs_fonts.ps	2024-03-11 20:29:48.649125943 +0100
@@ -142,10 +142,20 @@ userdict /.nativeFontmap .FontDirectory
       } if
       1 index type /stringtype eq
       1 index type /nametype eq and 1 index xcheck and
-      1 index /run eq 2 index /.runlibfile eq or and {
+      1 index /run eq 2 index /.runlibfile eq 3 index /.runlibfileifexists eq or or and {
                 % This is an inclusion entry.
-        pop findlibfile { exch pop } { file } ifelse
-        2 index exch .readFontmap pop
+        
+        0 index /.runlibfileifexists eq {
+          pop findlibfile {
+            exch pop
+            2 index exch .readFontmap pop
+          } {
+            pop
+          } ifelse
+        } {
+          pop findlibfile { exch pop } { file } ifelse
+          2 index exch .readFontmap pop
+        } ifelse
       } {
                 % This is a real entry.
                 % Read and pop tokens until a semicolon.
diff -up ghostpdl-10.03.0/Resource/Init/gs_init.ps.2~ ghostpdl-10.03.0/Resource/Init/gs_init.ps
--- ghostpdl-10.03.0/Resource/Init/gs_init.ps.2~	2024-03-11 20:29:48.649125943 +0100
+++ ghostpdl-10.03.0/Resource/Init/gs_init.ps	2024-03-11 20:45:09.567260091 +0100
@@ -894,6 +894,14 @@ currentdict /.makeinternaldict .undef
            { /undefinedfilename signalerror }
           ifelse
         } .internalbind def
+/runlibfileifexists
+        {		% We don't want to bind 'run' into this procedure,
+                        % since run may get redefined.
+          findlibfile
+           { exch pop /run .systemvar exec }
+           { pop }
+          ifelse
+        } bind def
 /selectdevice
         { finddevice setdevice .setdefaultscreen } .internalbind odef
 /signalerror		% <object> <errorname> signalerror -
@@ -1129,6 +1137,7 @@ def
         } .internalbind def
 % Temporarily substitute it for the real runlibfile.
 /.runlibfile /runlibfile load def
+/.runlibfileifexists /runlibfileifexists load def
 /runlibfile //runlibfile0 def
 currentdict /runlibfile0 .undef
 
